packages/@apphosting/adapter-nextjs/src/bin/build.spec.ts

```diff
@@ -97,6 +98,10 @@ metadata:
       },
       path.join(tmpDir, ".next"),
       defaultNextVersion,
+      {
```

Comment:
Since this shows up a couple times, could you preinitialize it in the beforeeach section?
For example:

adapterMetadata = {  adapterPackageName: "@apphosting/adapter-nextjs",
        adapterVersion: "14.0.1",
      },
 
 Then 
 await generateBuildOutput(
      tmpDir,
      tmpDir,
      outputBundleOptions,
      path.join(tmpDir, ".next"),
      defaultNextVersion,
      adapterMetadata
 }